src/iexamine.cpp

```diff
@@ -3516,6 +3565,11 @@ void iexamine::autodoc( player &p, const tripoint &examp )
 
     amenu.query();
 
+    // Pop-up messages ted to be of the form "You have $SOMETHING" or "NPC has $SOMETHING"
+    //~ Put the You have / NPC has section into a substring that only needs to be translated once
+    std::string pronoun = ( patient.name == p.name ? "You have " :
+                                                     string_format( _( "%1$s has "), patient.name ) );
```

Comment:
I don't find this sort of construct appropriate. Other languages may have another word ordering. It would be better to replace `popup` with `add_msg_if_player_or_npc` and add two variants of messages, for the sake of convenience of translation.

Reply:
Removed.